Lib/fontTools/merge.py

```diff
@@ -993,20 +1078,39 @@ def merge(self, fontfiles):
 			A :class:`fontTools.ttLib.TTFont` object. Call the ``save`` method on
 			this to write it out to an OTF file.
 		"""
-		mega = ttLib.TTFont()
+		# Take first input file sfntVersion
+		sfntVersion = ttLib.TTFont(fontfiles[0]).sfntVersion
+		mega = ttLib.TTFont(sfntVersion=sfntVersion)
 
 		#
 		# Settle on a mega glyph order.
 		#
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+
+		cffTables = []
+		if sfntVersion == "OTTO":
+			for i, font in enumerate(fonts):
+				_desubtoutinize(font)
+				cffTables.append(font['CFF '])
+
 		glyphOrders = [font.getGlyphOrder() for font in fonts]
-		megaGlyphOrder = self._mergeGlyphOrders(glyphOrders)
+
+		# Handle glyphOrder merging and cff glyphs renaming together
```

Comment:
maybe add a little comment as to why one has to "handle glyphOrder merging and cff glyphs renaming together"